From 763fcca5d1f742775c6ce103f39ba01b014634ee Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@marvell.com>
Date: Fri, 19 Jun 2020 02:17:13 -0700
Subject: [PATCH 1181/1239] Revert "configs: 98xx: Disable network driver"

This reverts commit 9a3e691fd2beb338b5d760dafe50eaf11bc1c55e.
ATF fix for RVU init Change-Id: Ib108138b037d179e4125341490925b4c83af8f5a.

Change-Id: If92c5555bec4330b1e2b5493ad0ab424db98c532
Signed-off-by: Suneel Garapati <sgarapati@marvell.com>
Depends-On: https://sj1git1.cavium.com/c/IP/SW/boot/atf/+/30365
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/boot/u-boot/+/30366
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
---
 configs/octeontx2_98xx_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/octeontx2_98xx_defconfig b/configs/octeontx2_98xx_defconfig
index aca267cbb6..0f6bbc50c0 100644
--- a/configs/octeontx2_98xx_defconfig
+++ b/configs/octeontx2_98xx_defconfig
@@ -97,6 +97,7 @@ CONFIG_DM_ETH=y
 CONFIG_E1000=y
 CONFIG_E1000_SPI=y
 CONFIG_CMD_E1000=y
+CONFIG_NET_OCTEONTX2=y
 CONFIG_NVME=y
 CONFIG_PCI=y
 CONFIG_DM_PCI=y
-- 
2.29.0

